src/styles/blocks/tags.css

```
@@ -0,0 +1,54 @@
+.tags__header {
+    margin: 0 0 37px;
+    font: 400 24px 'Dewi Expanded', sans-serif;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+.tags__list {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 -5px;
+    padding: 0;
+}
+
+.tags__item {
+    list-style: none;
+    margin: 0 5px 10px;
+}
+
+.tags__tag-link {
+    display: inline-block;
+    padding: 0 10px;
+    font-size: 16px;
+    line-height: 1.5;
+    font-weight: 400;
+    letter-spacing: 0.02em;
+    color: var(--color-cpecial-dark);
+}
+
+.tags__tag-link:hover:not(.tags__tag-link--active),
+.tags__tag-link:focus:not(.tags__tag-link--active) {
```

```suggestion
.tags__tag-link[href]:hover,
.tags__tag-link[href]:focus {
```
А может вот так будем проверять текущую ссылку?